tracee-ebpf/tracee/process_tree.go

```diff
@@ -0,0 +1,173 @@
+package tracee
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/aquasecurity/tracee/pkg/containers"
+)
+
+type ProcessCtx struct {
+	StartTime   int
+	ContainerID string
+	Pid         uint32
+	Tid         uint32
+	Ppid        uint32
+	HostTid     uint32
+	HostPid     uint32
+	HostPpid    uint32
+	Uid         uint32
+	MntId       uint32
+	PidId       uint32
+}
+
+type ProcessTree struct {
+	processTreeMap map[int]ProcessCtx
+}
+
+func (t *Tracee) ParseProcessContext(ctx []byte) (ProcessCtx, error) {
+	var procCtx = ProcessCtx{}
+	procCtx.StartTime = int(binary.LittleEndian.Uint64(ctx[0:8]))
+	cgroupId := binary.LittleEndian.Uint64(ctx[8:16])
+	procCtx.ContainerID = t.containers.GetCgroupInfo(cgroupId).ContainerId
+	dataBuff := bytes.NewBuffer(ctx[16:]) // this is the offset after the cgroup and startTime in the ctx byte array
```

Comment:
Can you please explain why do you read the rest of ctx?
And why you don't initialize the other fields of procCtx?